views/js/runner/provider/qti.js

```diff
@@ -253,6 +207,23 @@ define([
                 }, responses);
             }
             return responses;
+        },
+
+        renderFeedbacks : function(feedbacks, itemSession, done) {
+            var self = this;
+
+            var _renderer = self._item.getRenderer();
+            var _loader   = new QtiLoader(self._item);
+
+            // loading feedbacks from response into the current item
+            _loader.loadElements(feedbacks, function (item) {
+                _renderer.load(function () {
+
+                    var renderingQueue = modalFeedbackHelper.getFeedbacks(item, itemSession);
```

Comment:
feedback.render() - create html from object 
modalFeedbackHelper.getFeedbacks() - creates objects with information how and where to insert feedback